requirements/edx/kernel.in

```diff
@@ -159,3 +159,4 @@ webob
 web-fragments                       # Provides the ability to render fragments of web pages
 XBlock[django]                      # Courseware component architecture
 xss-utils                           # https://github.com/edx/edx-platform/pull/20633 Fix XSS via Translations
+slumber
```

Comment:
Could you also add a comment about the fact that this dependency is unsupported and used by the DeprecatedRestApiClient?